YoutubeExplode/Playlists/PlaylistClient.cs

```diff
@@ -143,12 +150,20 @@ public class PlaylistClient
                 );
 
                 videos.Add(video);
+
+                
             }
 
+            lastVideoId = playlistExtractor.TryGetLastVideoId();
+            index = playlistExtractor.TryGetLastIndex() ?? 0;
+            visitorData  = visitorData ?? playlistExtractor.TryGetVisitorData();
```

Comment:
What does `visitorData` actually look like?

Reply:
```"visitorData": "CgtOM25XYWtXNmJBTSis3ISPBg%3D%3D"```
Seems to be like a token, we need to use it  along with the index and lastVideoId if we want to get the next videos on the playlists.

Reply:
And visitorData only comes in the first response,  if we didn't use it responses would have all the time the same videos even if we changed the index and lastVideoId (at least for mix playlists)).